monkestation/code/game/objects/items/trash.dm

```diff
@@ -0,0 +1,3 @@
+/obj/item/trash/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_TRASH_ITEM, INNATE_TRAIT)
```

Comment:
I'm giving things that are already in the cleaner slime target typecache the trait, just so this trait can be useful for other things in the future